packages/xo-server-test/README.md

```diff
@@ -89,6 +89,8 @@ describe("user", () => {
 
 ## Usage
 
+We run the tests **sequentially** because some asynchronous tests can cause concurrency problems for some resources (servers, ...).  
```

Comment:
Tell me if it is not necessary to precise that.

Reply:
I think a simple comment in the configuration bellow the `concurrency` is enough.

Reply:
`comments are not permitted in JSON`

Reply:
Right :+1: 

The `Usage` section describe how to use `xo-server-test` , so, this message is not related to the section.